### PR TITLE
Align dash tests to support DPU with one single port

### PR DESCRIPTION
**Pull request by JibinBao**
### Description of PR

Align the existing dash tests to support DPU with one single port.
1.  For DPU with 2 ports, we have 2 neighbors, the first one is local PA, and the second one is remote PA. But for DPU with only one single port, we just have one neighbor (neighbor 1). So, we take neighbor 1 as the local PA. For the remote PA, we take the original neighbor 2's IP  as the remote PA IP,  and the original neighbor 2's network as the remote PA network. Take the mac of neighbor 1's mac as the mac of remote PA, because the BGP route to neighbor 1 is the default route, and only the mac of neighbor 1 exists in the arp table.  The remote ptf intf will take the value of neighbor 1 because the packet to remote PA will be forwarded to the ptf port corresponding to neighbor 1.
2.  Since some hwskus have the default acl rule action deny, we need to add the default acl rule action permit for them

Summary:
Fixes # (issue)

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311

#### What is the motivation for this PR?
Align the existing dash tests to support DPU with one single port.

#### How did you do it?
1.  For DPU with 2 ports, we have 2 neighbors, the first one is local PA, and the second one is remote PA. But for DPU with only one single port, we just have one neighbor (neighbor 1). So, we take neighbor 1 as the local PA. For the remote PA, we take the original neighbor 2's IP  as the remote PA IP,  and the original neighbor 2's network as the remote PA network. Take the mac of neighbor 1's mac as the mac of remote PA, because the BGP route to neighbor 1 is the default route, and only the mac of neighbor 1 exists in the arp table.  The remote ptf intf will take the value of neighbor 1 because the packet to remote PA will be forwarded to the ptf port corresponding to neighbor 1.
2.  Since some hwskus have the default acl route action deny, we need to add the default acl route action permit for them

#### How did you verify/test it?
run dash tests on dpu with 1 ports and 2 ports

#### Any platform specific information?
dpu

### CODE REVIEW

**Comment by JibinBao** [comment]
Hi @Kumaresh Perumal
Can you help review it?

**Review by nhe-NV — APPROVED** [review]

**Comment by JibinBao** [comment]
These failures are not related to my change.
Who can help check the failures?

**Comment by liat-grozovik** [comment]
@prsunny FYI in case you wish to review

**Review by oleksandrivantsiv — APPROVED** [review]

**Review by prsunny — APPROVED** [review]

**Review by kperumalbfn — APPROVED** [review]

**Review by prsunny — APPROVED** [review]

File: tests/dash/dash_acl.py [hunk 1]
```diff
@@ -155,7 +155,7 @@ def config(self):
             ACL_PRIORITY: 255,
             ACL_ACTION: self.default_action,
             ACL_TERMINATING: "false",
-            ACL_PROTOCOL: "17",
+            ACL_PROTOCOL: "17, 6, 1, 2",
```
Comment (kperumalbfn) [inline tests/dash/dash_acl.py:158]: why IGMP required?

Comment (JibinBao) [reply tests/dash/dash_acl.py]: You are right. We don't need it. I have removed it.
Thanks. Please help review it again.

**Review by kperumalbfn — APPROVED** [review]

**Comment by JibinBao** [comment]
Hi @prsunny @kperumalbfn ,
If there is no new comment, can you help merge it?

**Review by liat-grozovik — APPROVED** [review]

**Pull request merged**